<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditeDialog</class>
 <widget class="QDialog" name="EditeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>731</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Bebas Neue Regular</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nome da receita</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_nome_receita"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Bebas Neue Regular</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>LINK DA RECEITA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_link"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Bebas Neue Regular</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Grau de dificuldade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_dificuldade">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fácil</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Médio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Díficil</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Profissional</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Bebas Neue Regular</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Categoria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_categoria">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Entrada</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Prato Principal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sobremesa</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aperitivo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Acompanhamento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lanche</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Salada</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Molho</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Bebas Neue Regular</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Estrelas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputEstrelas"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="cadastrar">
        <property name="text">
         <string>cadastrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelar">
        <property name="text">
         <string>cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
